source/application/AMPIC887DriverServer/AMPIC887AxisMap.h

```diff
@@ -0,0 +1,297 @@
+#ifndef AMPIC887AXISMAP_H
+#define AMPIC887AXISMAP_H
+
+#include <QMap>
+#include <QList>
+#include "AMGCS2.h"
+#include "AMPIC887AxisCollection.h"
+#include "AMGCS2Support.h"
+#include "util/AMCArrayHandler.h"
+/*!
+  * A class which maps Axes of a PI C887.11 controller to a type T.
+  */
+template <typename T>
+class AMPIC887AxisMap : public QMap<AMGCS2::Axis, T>
+{
+public:
+	/*!
+	  * Creates an instance of an axis map, which is empty.
+	  */
+	AMPIC887AxisMap() {}
```

Comment:
The same result will be produced either way 
http://stackoverflow.com/questions/5780093/c-should-i-call-the-base-class-default-constructor-in-the-initialization-li
I've added it in anyway for clarity.